src/teleop/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -51,13 +51,11 @@
         </div>
         <div class="stats">
           <p>
-            Waypoints Traveled: {{nav_status.completed_wps}}/{{nav_status.total_wps}}<br>
+            Velocity Commands<br>
```

Comment:
Could we maybe make this a component instead of having it loose in the waypoint editor page? This will be a useful thing to log elsewhere as well.